spec/rails_helper.rb

```diff
@@ -69,21 +77,24 @@
 
   # This should exclude all specs with metadata "swagger: true"
   config.filter_run_excluding swagger: true
+
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.around do |example|
+    DatabaseCleaner.cleaning do
+      example.run
+    end
+  end
 end
 
 def seed_live_data
-  Dir[Rails.root.join("db/populators/*.rb")].sort.each do |seed_file|
-    require seed_file
+  # you get lots of <class> or <constant> already initialized/defined warnings without this silencer
+  Kernel.silence_warnings do
+    load Rails.root.join("db/seeds.rb")
```

Comment:
Can this guarantee sequence? 
I know we have had issues in the past with can't seed X because Y isn't populated yet

Reply:
not sure, although the sequence here will be exactly what the running app would on pod spin up (`docker_scripts/setup_db`, do so I'd hope so 😬 . To me this has the added benefit of breaking tests if the order is wrong for the app seeding.

Reply:
Discussed and happy to merge as is and monitor :+1: